src/scss/_global/_structure.scss

```diff
@@ -3,6 +3,8 @@ body {
   display: flex;
   flex-direction: column;
   justify-content: flex-start;
+  overflow-x: hidden;
+  overflow-y: scroll;
```

Comment:
This shouldn't be necessary unless you've encountered a bug where page content should've been scrollable but wasn't.